shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -263,7 +266,20 @@ - (void)installSplashScreenViewIfNecessary {
   [self.view addSubview:splashScreenView];
 }
 
++ (BOOL)automaticallyNotifiesObserversOfIsDisplayingFlutterUI {
+  return NO;
+}
+
+- (void)setIsDisplayingFlutterUI:(BOOL)isDisplayingFlutterUI {
+  if (_isDisplayingFlutterUI != isDisplayingFlutterUI) {
+    [self willChangeValueForKey:@"isDisplayingFlutterUI"];
+    _isDisplayingFlutterUI = isDisplayingFlutterUI;
+    [self didChangeValueForKey:@"isDisplayingFlutterUI"];
+  }
+}
+
 - (void)callViewRenderedCallback {
+  self.isDisplayingFlutterUI = YES;
```

Comment:
I hate to keep harping on this PR, but did we figure out if it's a problem that this isn't set in a queue-confined way to guarantee YES -> NO -> YES -> NO?

Reply:
No - I think the bigger problem is that we're not guaranteed to call this accurately though.  I'll think about this some more.

Reply:
fixed in c6923c6